update.go

```diff
@@ -79,14 +79,35 @@ func (m *Manager) InstallTo(path, dir string) error {
 
 	dst := filepath.Join(dir, m.Command)
 
-	log.Debugf("move %q to %q", bin, dst)
-	if err := os.Rename(bin, dst); err != nil {
-		return errors.Wrap(err, "moving")
+	log.Debugf("copy %q to %q", bin, dst)
+
+	if err := copy(bin, dst); err != nil {
+		return errors.Wrap(err, "copy")
 	}
 
 	return nil
 }
 
+func copy(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
```

Comment:
probably best to check and return errors from these, I think `defer Close()` is kind of an anti-pattern, we'd like to know about those errors

Reply:
We could use https://godoc.org/github.com/pkg/fileutils and delegate that stuff

Reply:
Good idea